### PR TITLE
Estonian and Portuguese languages update

### PR DESCRIPTION
**Pull request by jphms**
Just a small update to use Estonian locale in Carbon 2.

### CODE REVIEW

**Comment by kylekatarnls** [comment]
Thanks a lot! 😎

**Comment by kylekatarnls** [comment]
Could you update https://github.com/jphms/Carbon/blob/master/tests/Localization/EtTest.php with the actual translations?

File: src/Carbon/Lang/pt.php [hunk 1]
```diff
@@ -28,6 +28,15 @@
     'from_now' => 'em :time',
     'after' => ':time depois',
     'before' => ':time antes',
+    'diff_now' => 'agora',
+    'diff_yesterday' => 'ontem',
+    'diff_tomorrow' => 'amanhã',
+    'diff_before_yesterday' => 'anteontem',
+    'diff_after_tomorrow' => 'depois de amanhã',
```
Comment (kylekatarnls) [inline src/Carbon/Lang/pt.php]: Are you sure `anteontem` and mostly `depois de amanhã` are commonly used? If not, we should rather not translate them and so fallback to calendar translations.

**Comment by jphms** [comment]
Yes, I'll do that. 🤟

File: src/Carbon/Lang/pt.php (hunk 1 above)
Comment (kylekatarnls) [reply src/Carbon/Lang/pt.php]: Thanks.

**Comment by jphms** [comment]
> Could you update https://github.com/jphms/Carbon/blob/master/tests/Localization/EtTest.php with the actual translations?

Done. 👍

**Pull request merged**